tutorials/create-models-django/index.mdx

```diff
@@ -10,12 +10,10 @@ categories:
 tags: compute installing Django-application Models API
 hero: assets/scaleway_django_models.webp
 dates:
-  validation: 2023-08-28
+  validation: 2024-03-05
   posted: 2018-11-02
 ---
 
-## Django Overview
-
 Django is a web development framework for developing dynamic websites and applications. Django allows the use of less code and eases the creation of complex websites. It provides a very good structure and easy methods that help to do the heavy lifting when writing web applications.
 
 To follow this tutorial, we assume that you completed the [first tutorial on Django installation and configuration](https://www.scaleway.com/docs/how-to-install-django-web-framework/).
```

Comment:
I don't think the link is right here, the one I am suggesting looks like it could match the one you are trying to link.
```suggestion
To follow this tutorial, we assume that you completed the [first tutorial on Django installation and configuration](/tutorials/django-ubuntu-focal-fossa/).
```